plugins/generator-1.15.2/forge-1.15.2/templates/block.java.ftl

```diff
@@ -173,6 +173,12 @@ public class ${name}Block extends ${JavaModName}Elements.ModElement {
 					<#if data.slipperiness != 0.6>
 					.slipperiness(${data.slipperiness}f)
 					</#if>
+					<#if data.speedFactor != 1.0>

```

Comment:
I would still omit these parameters at the default value so they are only added to the code when needed

Reply:
I am maybe wrong, but they are already only added if the value is not equal to 1.9 (so the default value).

Reply:
But default value in UI is now different

Reply:
I don't know if you wanted to say that the default value in UI was 1 instead of 1.0, but it worked perfectly.